CHANGELOG.md

```diff
@@ -11,6 +11,14 @@
 - Breaking changes
   - Remove `Context` parameter from `CriteoInterstitial` constructor
   - Remove `Parcelable` implementation from `AdSize` class
+  - Move `CriteoInterstitialAdDisplayListener#onAdFailedToDisplay(CriteoErrorCode)` to
+   `CriteoInterstitialAdListener#onAdFailedToReceive(CriteoErrorCode)`. This callback is fired
+    when any kind of error happen after requesting an interstitial ad.
```

Comment:
This callback is fired when an error happens after requesting an interstitial ad. 

(Any should be used only for uncountable nouns and plurals and when the sentence is a question or a negative).